javascript/chronometer.js

```diff
@@ -1,26 +1,53 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = "";
+    //this.milliseconds = 0;
+    //this.intervalIdMilli = "";
   }
   startClick(callback) {
-    // ... your code goes here
-  }
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1
+      callback();
+    }, 1000)
+    // this.intervalId = setInterval(() => {
+    //   this.milliseconds += 1
+    //   callback();
+    // }, 100)
+  }
+
   getMinutes() {
-    // ... your code goes here
+    return this.twoDigitsNumber(Number(Math.floor(this.currentTime / 60)));
   }
   getSeconds() {
-    // ... your code goes here
-  }
-  twoDigitsNumber() {
-    // ... your code goes here
+    if (!this.currentTime) {
+      return 0
+    } else {
+      return this.twoDigitsNumber(Number(this.currentTime % 60));
+    }
+  }
+  // getMilliseconds() {
+  //   return this.twoDigitsNumber(Number(this.milliseconds));
+  // }
+  twoDigitsNumber(timeNumber) {
+    if (timeNumber < 10) {
+      return String(`0${timeNumber}`);
+    } else {
+      return String(timeNumber);
+    }
   }
   stopClick() {
-    // ... your code goes here
+    clearInterval(this.intervalId);
+    //clearInterval(this.intervalIdMilli)
   }
   resetClick() {
-    // ... your code goes here
+    this.currentTime = 0;
+    //this.milliseconds = 0;
   }
   splitClick() {
-    // ... your code goes here
+    if (!this.currentTime) {
+      return '00:00'
+    }
```

Comment:
Jasmine will pass here as the test is with the 0 time split, however the idea was to put here the functionality to return the correct format of the current time, to then be used in the index.js file. However I noticed the functionality is there and this method is never being used.